### DISCUSSION TITLE
Github Copilot Version Update

### DISCUSSION THREAD

**Discussion by szaveri3** [community/community#41924] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi! My github copilot is not working, and I can't seem to pinpoint why. I have updated and installed the most current version I can install, but for some reason the output still says my version is out of date. Any ideas on how to fix?

<img width="637" alt="Screen Shot 2022-12-16 at 11 21 12 AM" src="https://user-images.githubusercontent.com/79428339/208143291-29a0d387-72c6-4954-a6eb-0b9300598a08.png">

**Comment by Alan-eMartin** [comment]
Hey there, I am also having the same issue after updating.

I am authenticated the extension is installed.
I have restarted my vs code, computer, everything.

No Luck

**Comment by cinderjk** [comment]
Same here

**Reply by maherkhalil07** [reply]
same here

**Comment by ddematheu2** [comment]
Can confirm. Just rolled back version and things seem to be working again.

**Reply by CoreyMcCoy** [reply]
Perfect...TU

**Reply by szaveri3** [reply]
what version did you roll back to?

**Reply by franklin-tina** [reply]
@szaveri3 try **1.62.7587**.
By the way, I'm using the nightly version

**Reply by franklin-tina** [reply]
I can confirm the latest update works—not on the nightly version though.

**Comment by iamcjwilliams** [comment]
@szaveri3 Sorry for the trouble here, please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-github-copilot-in-visual-studio-code
